0-common/schema-ddl/src/main/scala/com.snowplowanalytics/iglu.schemaddl/sql/AlterTable.scala

```diff
@@ -78,7 +80,7 @@ case class AddColumn(
   columnName: String,
   columnType: DataType,
   default: Option[Default],
-  encode: Option[CompressionEncoding],
+  encode: Option[ColumnAttribute],
```

Comment:
Because regular sql dbs do not have compression encoding, but my have other attributes.